src/components/Counter.js

```diff
@@ -1,39 +1,40 @@
 import React, { Component } from "react";
 import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { increment, decrement, incrementIfOdd, incrementAsync  } from '../actions';
 
 class Counter extends Component {
-    incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
-    };
+    // incrementIfOdd = () => {
+    //     Stretch Problem: Implement an increment function that
+    //     only increments if the counter value is odd
+    // };
 
-    incrementAsync = () => {
-        // Stretch Problem: Implement an increment function that
-        // increments after waiting for one second
-    };
+    // incrementAsync = () => {
```

Comment:
Likewise, you could use the same increment action creator and use `setTimeout()` to invoke it after one second